src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6034,4 +6037,25 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_BadFuncPointerArgCount" xml:space="preserve">
     <value>Function pointer '{0}' does not take {1} arguments</value>
   </data>
+  <data name="ERR_MethFuncPtrMismatch" xml:space="preserve">
+    <value>No overload for '{0}' matches function pointer '{1}'</value>
+  </data>
+  <data name="ERR_FuncPtrRefMismatch" xml:space="preserve">
+    <value>Ref mismatch between '{0}' and function pointer '{1}'</value>
+  </data>
+  <data name="ERR_FuncPtrMethMustBeStatic" xml:space="preserve">
+    <value>Cannot create a function pointer for '{0}' because it is not a static method</value>
+  </data>
+  <data name="ERR_AddressOfMethodGroupInExpressionTree" xml:space="preserve">
+    <value>'&amp;' on method groups cannot be used in expression trees</value>
+  </data>
+  <data name="ERR_WrongFuncPtrCallingConvention" xml:space="preserve">
+    <value>Calling convention of '{0}' is not compatible with '{1}'.</value>
+  </data>
+  <data name="ERR_MissingAddressOf" xml:space="preserve">
+    <value>Cannot convert method group to function pointer (Are you missing a '&amp;'?)</value>
+  </data>
+  <data name="ERR_CannotUseReducedExtensionMethodInAddressOf" xml:space="preserve">
+    <value>Cannot use a reduced extension method as the target of a '&amp;' operator.</value>
```

Comment:
>reduced [](start = 24, length = 7)

I don't think language specification uses the "reduced" term, it probably came from VB. Perhaps we should say: "Cannot use a an extension method with a receiver as the target of a '&amp;' operator."